middleware/filament_usage.py

```
@@ -146,6 +146,50 @@ def _publish_deduction(device_id: str, uid: str, deduct_g: float) -> None:
         logger.exception("UPDATE_TAG: failed to publish deduction")
 
 
+LOW_SPOOL_HYSTERESIS_G: float = 50.0
+
+
+def _publish_low_spool(device_id: str, latched: bool) -> None:
```

Ruff (0.15.10)

[warning] Boolean-typed positional argument in function definition

(FBT001)